landing-pages/site/content/en/blog/airflow-survey.md

```diff
@@ -0,0 +1,333 @@
+---
+title: "Airflow Survey 2019"
+linkTitle: "Airflow Survey 2019"
+author: "Tomek Urbaszek"
+twitter: "Nuclearriot"
+github: "nuclearpinguin"
+linkedin: "tomaszurbaszek"
+description: "Receiving and adjusting to our users’ feedback is a must. Let’s see who Airflow users are, how they play with it and what they miss."
+tags: ["community", "survey", "users"]
+date: "2019-12-11"
+---
+# Apache Airflow Survey 2019
+
+Apache Airflow is [growing faster than ever](https://www.astronomer.io/blog/why-airflow/).
+Thus, receiving and adjusting to our users’ feedback is a must. Let’s see who Airflow
+users are, how they play with it and what they miss. In this survey we got **308** responses.
+The original form can be found here:
+https://forms.gle/XAzR1pQBZiftvPQM7
+
+# Overview of the user
+
+**What best describes your current occupation?**
+
+|                         |No.|  %   |
+|-------------------------|---|------|
+|Data Engineer            |194|62.99%|
+|Developer                | 34|11.04%|
+|Architect                | 23|7.47% |
+|Data Scientist           | 19|6.17% |
+|Data Analyst             | 13|4.22% |
+|DevOps                   | 13|4.22% |
+|IT Administrator         |  2|0.65% |
+|Machine Learning Engineer|  2|0.65% |
+|Manager                  |  2|0.65% |
+|Operations               |  2|0.65% |
+|Chief Data Officer       |  1|0.32% |
+|Engineering Manager      |  1|0.32% |
+|Intern                   |  1|0.32% |
+|Product owner            |  1|0.32% |
+|Quant                    |  1|0.32% |
+
+
+**In your day to day job, what do you use Airflow for?**
+
+|                                                      |No.|  %   |
+|------------------------------------------------------|---|------|
+|Data processing (ETL)                                 |298|96.75%|
+|Artificial Intelligence and Machine Learning Pipelines| 90|29.22%|
+|Automating DevOps operations                          | 64|20.78%|
+
+According to the survey most of Airflow users are the “data” people. Moreover,
+28.57% uses Airflow to both ETL and ML pipelines meaning that those two fields
+are somehow connected. Only 5 people use Airflow for DevOps operations only,
+this suggests that if Airflow is used by someone then he or she will also try it
+for DevOps tasks.
```

Comment:
I'm not sure I follow you here.
5 people use airflow for devops only ✅ 
this doesn't imply that all the other respondents use it for DevOps and something else . A respondent can use it for ETL and ML and not DevOps

Reply:
I think it implies that because 64 users selected "devops" but 5 of them selected only this option meaning that the rest, 55 users uses Airflow for both devops and ETL / ML. I can agree that direction of this implication is only an assumption based on results.

Reply:
Ah ok I understand what you meant: so "if you use it for DevOps, you don't use it only for DevOps". But there are still 308-64 people that don't care about DevOps so to speak

Reply:
I 'd also remove the last sentence :) it's not obvious.

Reply:
I reformulated it